src/components/screens/editVote/index.js

```diff
@@ -97,11 +95,11 @@ const AddVote = ({
             <p className={styles.balanceTitle}>{t('Available balance')}</p>
             <div className={styles.balanceDetails}>
               <span className={styles.lskValue}>
-                <LiskAmount val={balance} token={tokenMap.LSK.key} />
+                <LiskAmount val={maxAmount} token={tokenMap.LSK.key} />
```

Comment:
this logic will replace the account balance? e.g it will only display whats the maximum possible amount to send rather than balance available. The modal shows `Available balance` and we should display `Available balance`

Reply:
@ManuGowda That's one of the few ways to fix the issue as it was described but I'll align with @reyraa to find the best solution